apps/backend/src/excelexport/excelexport.service.ts

```diff
@@ -53,20 +54,60 @@ export class ExcelexportService {
     }
   }
 
+  public generateWorkbook(title: string): void {
+    this.workbook.title = title;
+    this.workbook.created = new Date(format(new Date(), 'yyyy, MM, dd'));
+  }
+
   public addSheet(name: string): void {
     this.workbook.addWorksheet(name);
   }
 
+  /* eslint-disable  */
+  public addHeader(sheetName: string, style: any): void {
+    const sheet = this.workbook.getWorksheet(sheetName);
+    sheet.insertRow(1, []);
+    const row = sheet.getRow(1);
+    row.addPageBreak();
+    row.getCell('E').value = sheetName;
```

Comment:
determine cell / cells based on input

Reply:
resolved in report_two

Reply:
Passing in a variable from the reporting service to set the header placement